build/dockerfiles/entrypoint.sh

```diff
@@ -44,22 +44,6 @@ if [[ "$1" == "kong" ]]; then
   if [[ "$2" == "docker-start" ]]; then
     kong prepare -p "$PREFIX" "$@"
 
-    # remove all dangling sockets in $PREFIX dir before starting Kong
```

Comment:
we will still need the logic, but instead removing dangling sockets in /var/run/kong directory.
I was wrong with my previous comment some days ago, it appears the `/run` or `/var/run` directory is not a volatile fs inside container; it's only volatile in a normal system. So the contents under this directory persists between container restarts.